Automattic/jetpack
# Pocket Casts: Migrate over as core/embed block variation extension.

This PR is current **ON HOLD** until Phase 2 (pdDOJh-bf-p2) starts.

#### Changes proposed in this Pull Request:
This PR migrates the "non-block" Pocket Casts block variation to `extensions/extended-blocks/core-embed`.
More explanation in the PR https://github.com/Automattic/jetpack/pull/23740.

#### Jetpack product discussion
pdDOJh-bf-p2 & pdeCcb-dg-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

1. Open up block editor.
2. Add "Pocket Casts" block to your editor.
3. Paste a Pocket Cast link.
4. It should embed a Pocket Cast player.

![image](https://user-images.githubusercontent.com/1287077/161160385-75773707-77d1-490c-b7bd-db8b45a78d0b.png)

![image](https://user-images.githubusercontent.com/1287077/161160482-072127aa-44e3-4a5d-aff9-447ea8944488.png)

Fixes #
